### DISCUSSION TITLE
Menu color only changes the color of the big buttons

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#253] [Q&A] [answered]
Application --> menu color only changes the color of the big buttons, the rest stays in blue. Can this be changed that also the smaller symbols of the menu and also the back and forth arrow gets the same color as this setting? 

<img src="https://github.com/devemux86/cruiser/assets/121371629/4646a41a-32e2-4a58-9fe0-683a632ceb83" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/121371629/f351e683-3c12-46eb-8751-b1fe4b4dfc5f" width="33%"/>

**Comment by devemux86** [comment]
The Settings and most app areas use the app's accent color.
Unfortunately Android does not allow to change it. I want to search more if it is possible at all.

> Can this be changed that also the smaller symbols of the menu and also the back and forth arrow gets the same color as this setting?

@DolomitesTom I will check if it is possible for these buttons.

---

I asked in the past if we really need the "Menu color" in its current state?
(works only for menu buttons)

**Reply by WolfgangZ42** [reply]
> if we really need the "Menu color"

for my opinion it's not necessary

**Comment by devemux86** [comment]
Preview of next Cruiser:

<img src="https://github.com/devemux86/cruiser/assets/3484020/7e0467b4-8217-4a37-96f0-69796809ea89" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/a9146361-0224-461c-9f6b-1557d906ec45" width="50%"/>

**Reply by DolomitesTom** [reply]
How do you get the little icons and the arrow also in yellow?

**Reply by devemux86** [reply]
@DolomitesTom I forgot to mention that it is a preview of the next Cruiser. :slightly_smiling_face:

**Comment by devemux86** [comment, marked as answer]
> The Settings and most app areas use the app's accent color.
Unfortunately Android does not allow to change it. I want to search more if it is possible at all.

It is possible in Cruiser and will be available in the next Cruiser update.
(as a new setting that will replace the limited menu color)

- https://github.com/devemux86/cruiser/discussions/256
